tmt/options.py

```diff
@@ -270,6 +274,8 @@ def _check_method(self, context: 'tmt.cli.Context', args: List[str]) -> None:
                 # Stop search at the first argument looking like a subcommand
                 elif arg in subcommands:
```

Comment:
Isn't `is_likely_subcommand` all we need now?

Reply:
Nice catch! `elif arg in subcommands` is removed.